app/mixins/ads.js

```diff
@@ -26,6 +26,9 @@ export default Mixin.create({
 	init() {
 		this._super(...arguments);
 		this.renderAdComponent = getRenderComponentFor(this);
+
+		// Set proper UAP language
+		window.wgContentLanguage = this.get('wikiVariables.language.content');
```

Comment:
Maybe `context.set('options.contentLanguage')`?

Reply:
Actually, it would be enough to set this in the bridge (app repo) and close this pull request.

Reply:
I think you right. Closing.